apps/tlon-mobile/src/screens/ChannelScreen.tsx

```diff
@@ -225,7 +225,9 @@ export default function ChannelScreen(props: ChannelScreenProps) {
   const handleGroupAction = useCallback(
     async (action: GroupPreviewAction, updatedGroup: db.Group) => {
       if (action === 'goTo' && updatedGroup.lastPost?.channelId) {
-        const channel = await store.getChannel(updatedGroup.lastPost.channelId);
+        const channel = await db.getChannel({
```

Comment:
a `db` method called from a screen component *gasp*.  Not a huge deal, but feels weird to me

Reply:
Lol yaa there are a few of these -- I think store should re-export queries ultimately, but that's for a later pass.